wf-galleon/WFLY-19021-Stability_In_Provisioning.adoc

```diff
@@ -0,0 +1,165 @@
+= WildFly provisioning to support WildFly stability
```

Comment:
Please add Jekyll front matter at the start of this file so it will show up properly categorized in docs.widlfly.org/wildfly-proposals.  See (almost) any merged proposal for format. See _data/wildfly-categories.yaml for the categories.

Reply:
Done.